llvm/test/MC/EraVM/asm-parser/far-calls.s

```diff
@@ -25,12 +25,21 @@ foo:
   far_call.delegate.shard.lt    r3, r4, @handler
   far_call.mimic.shard.lt       r3, r4, @handler
 
-  far_call.{st,sh}           r3, r4, @handler
-  far_call.delegate.{st,sh}       r3, r4, @handler
-  far_call.mimic.{st,sh}          r3, r4, @handler
-  far_call.{st,sh}.lt        r3, r4, @handler
-  far_call.delegate.{st,sh}.lt    r3, r4, @handler
-  far_call.mimic.{st,sh}.lt       r3, r4, @handler
+  far_call.st.sh           r3, r4, @handler
```

Comment:
And for mnemonics, do we still accept the old notation?

Reply:
What is the old notation? Support for `.{st,sh}` notation is explicitly removed by this PR (replaced by `.st.sh` or `.sh.st`).

Considering the specific names used for mnemonics and modifiers, this PR uses the old mnemonic names (such as `far_call` instead of `callf`). It uses old modifier names for pre-existing modifiers as well (`.static` and `.shard`) to prevent CI failures. The combined `.st.sh` form is different as it did not exist before #539, that is why I use the new syntax even before renaming everything else.